lib/main_common.pm

```diff
@@ -939,9 +939,10 @@ sub load_toolchain_tests {
 }
 
 sub load_common_opensuse_sle_tests {
-    load_autoyast_clone_tests if get_var("CLONE_SYSTEM");
-    load_create_hdd_tests     if get_var("STORE_HDD_1") || get_var("PUBLISH_HDD_1");
-    load_toolchain_tests      if get_var("TCM") || check_var("ADDONS", "tcm");
+    load_autoyast_clone_tests  if get_var("CLONE_SYSTEM");
+    load_create_hdd_tests      if get_var("STORE_HDD_1") || get_var("PUBLISH_HDD_1");
+    load_toolchain_tests       if get_var("TCM") || check_var("ADDONS", "tcm");
+    loadtest "console/network" if get_var('NETWORK_CONFIGURATION');
```

Comment:
We should try to use single ticks by default everywhere so please also in `loadtest`